cloudsql/postgresql/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "appengine-cloudsql-postgres",
+  "description": "Node.js PostgreSQL sample for Cloud SQL on App Engine.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=8"
+  },
+  "scripts": {
+    "lint": "repo-tools lint",
+    "pretest": "npm run lint",
+    "unit-test": "ava --verbose test/*.test.js",
+    "start-proxy": "! pgrep cloud_sql_proxy > /dev/null && cloud_sql_proxy -instances=$INSTANCE_CONNECTION_NAME=tcp:$SQL_PORT &",
+    "system-test": "repo-tools test app -- server.js",
+    "system-test-proxy": "npm run start-proxy; npm run system-test",
+    "all-test": "npm run unit-test && npm run system-test",
+    "test": "repo-tools test run --cmd npm -- run all-test"
+  },
+  "dependencies": {
+    "async": "2.6.0",
+    "body-parser": "1.18.2",
+    "express": "4.16.2",
+    "knex": "0.14.4",
+    "pg": "7.4.1",
+    "prompt": "1.0.0",
+    "pug": "2.0.0-rc.4",
+    "@google-cloud/logging-winston": "^0.10.2",
+    "winston": "^3.1.0"
+  },
+  "devDependencies": {
+    "@google-cloud/nodejs-repo-tools": "2.2.1",
```

Comment:
Please update to version 3 of `repo-tools`

Reply:
Done.